workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandResponseHandler.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using Improbable.Gdk.Core.Commands;
+using Unity.Entities;
+using Entity = Unity.Entities.Entity;
+
+#region Diagnostic control
+
+// ReSharper disable UnusedMember.Local
+
+#endregion
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static partial class WorldCommandsRequirables
+    {
+        [InjectableId(InjectableType.WorldCommandResponseHandler, InjectableId.NullComponentId)]
+        [InjectionCondition(InjectionCondition.RequireNothing)]
+        public class WorldCommandResponseHandler : IInjectable
```

Comment:
Now that #288 has been merged, can you make this inherit `RequirableBase`?

Reply:
Aye